index.html

```diff
@@ -1,29 +1,36 @@
 <!DOCTYPE html>
 <html>
 
-<head>
-<meta charset="utf-8">
-<link href="style.css" rel="stylesheet">
-</head>
-
-<body>
-
-<h1>todos</h1>
-
-<div class="tasks">
-	<div class="task_new"><label> ˅ </label><em> What needs to be done?</em></div>
-	<div class="task_completed"><input type="checkbox" checked>to learn HTML</div>
-	<div class="task_active"><input type="radio">to learn CSS</div>
-	<div class="task_active"><input type="radio">to learn JS</div>
-
-	<div class="filters">
-		<div class="filter_quantity_tasks">2 items left</div>
-		<div class="filter_show bordered">All</div>
-		<div class="filter_show">Active</div>
-		<div class="filter_show">Completed</div>
-		<div class="filter_delete">Clear completed</div>
-	</div>
-</div>
-
-</body>
+	<head>
+		<meta charset="utf-8">
+		<link href="style.css" rel="stylesheet">
+	</head>
+
+	<body>
+
+		<h1>todos</h1>
+
+		<div class="tasks">
```

Comment:
naming 
think of hierarchy, maybe it'snot "tasks" any more?)

Reply:
changed